resources/deployments/controller/controller_deployments.go

```diff
@@ -65,7 +65,7 @@ func (d *DeploymentsController) PostDeployment(w rest.ResponseWriter, r *rest.Re
 		return
 	}
 
-	d.view.RenderSuccessPost(w, r, id)
+	d.view.RenderSuccessPost(w, r, "./deployments/"+id)
```

Comment:
well, the controller knows what resource is requested, not the view. the view is dumber than the controller, so it it needs to know what path exactly to set.